### PR TITLE
Added GKE optimization signals dashboard

### PR DESCRIPTION
**Pull request by aburhan**
Added additional gke namespace filters and switch to using max vs mean on queries

### CODE REVIEW

File: dashboards/google-kubernetes-engine/README.md [hunk 1]
```diff
@@ -46,3 +46,12 @@
 |:---------------------|
 |Filename: [gke-nodes-pods-cluster-view.json](gke-nodes-pods-cluster-view.json)|
 |This dashboard has 10 charts using the related [Kubernetes metrics](https://cloud.google.com/monitoring/api/metrics_kubernetes) to show how pods and containers are distributed within nodes and within namespaces. Container start, restart and autoscaler events are also plotted over time.|
+
+&nbsp;
+
+|GKE Optimization Golden Signals|
```
Comment (EvanSimpson) [inline dashboards/google-kubernetes-engine/README.md:52]: Can you add an entry for the new dashboard to the metadata.yaml file in the same directory?

**Comment by draffensperger** [comment]
It looks like these are CPU and memory metrics, which I would consider to be Saturation metrics (in https://sre.google/sre-book/monitoring-distributed-systems/). Since we lack the other three golden signals here of Latency, Traffic and Errors, would it be OK to avoid to "Golden Signals" name here and call it something else like "GKE Resource Usage" dashboard or something like that?

File: dashboards/google-kubernetes-engine/README.md (hunk 1 above)
Comment (aburhan) [reply dashboards/google-kubernetes-engine/README.md]: done

**Comment by aburhan** [comment]
> It looks like these are CPU and memory metrics, which I would consider to be Saturation metrics (in https://sre.google/sre-book/monitoring-distributed-systems/). Since we lack the other three golden signals here of Latency, Traffic and Errors, would it be OK to avoid to "Golden Signals" name here and call it something else like "GKE Resource Usage" dashboard or something like that?

Updated the title to GKE Optimization Signals. based on "State of Kubernetes Cost Optimization report"

File: dashboards/google-kubernetes-engine/gke-workloads-at-risk.json [hunk 2]
```diff
@@ -347,7 +352,7 @@
...
+                  "timeSeriesQueryLanguage": "fetch k8s_container\n| metric 'kubernetes.io/container/cpu/request_utilization'\n    | ${project_id}\n    | ${location}\n    | ${cluster_name}\n| filter resource.namespace_name !~ '(kube|istio|gatekeeper|gke|gmp|gke-gmp)-system|gke-managed-filestorecsi|gke-mcs'\n| every 5m\n| group_by\n    [resource.project_id, resource.location, resource.cluster_name,\n     resource.namespace_name,\n     metadata.system.top_level_controller_name:\n       metadata.system_labels.top_level_controller_name],\n    [value_request_utilization_mean:\n       mean(value.request_utilization)]\n| filter value_request_utilization_mean >= 1\n| scale '%'"
```
Comment (EvanSimpson) [inline dashboards/google-kubernetes-engine/gke-workloads-at-risk.json:355]: Are the changes to the namespace regexes intended to be included in this PR?

Comment (aburhan) [reply dashboards/google-kubernetes-engine/gke-workloads-at-risk.json:355]: Yes, I want to exclude system namespaces

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**